spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -111,11 +110,21 @@
 	 * @param maxMessages the maximum number of pulled messages
 	 * @param returnImmediately returns immediately even if subscription doesn't contain enough
 	 * messages to satisfy {@code maxMessages}
-	 * @param retrySettings the timeout and retry setting for the pull request
 	 * @return the list of received messages
 	 */
-	List<PubsubMessage> pull(String subscription, Integer maxMessages,
-			Boolean returnImmediately, RetrySettings retrySettings);
+	List<PubsubMessage> pullAndAck(String subscription, Integer maxMessages,
+			Boolean returnImmediately);
+
+	/**
+	 * Pull a number of messages from a Google Cloud Pub/Sub subscription.
+	 * @param subscription the subscription name
+	 * @param maxMessages the maximum number of pulled messages
+	 * @param returnImmediately returns immediately even if subscription doesn't contain enough
+	 * messages to satisfy {@code maxMessages}
+	 * @return the list of received (n)acknowledgeable messages
```

Comment:
I think the language in JavaDocs should be official, not such a formal like we may use in the source code.
I mean that I don't believe that there is such a `nacknowledgeable` word at all.
Forgive me if I am missing anything here.